students/PadmajaPeri/lesson03/mailroom.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+"""
+A simple program to keep track of donations for the charity. The program
+provides option to 1) Create report of all donations and donor details
+2) Send Thank You Notes 3) Add new donors 4) Query for donors
+__author__ = "Padmaja Peri"
+"""
+
+import sys
+
+# Pre-populate a list of donors
+donation_list = [['William Gates, III', 22000, 653784.49], ['Mark Zuckerberg',
+                    2500, 16396.10], ['Jeff Bezos', 800, 877.33], ['Larry Page'
+    , 1, 20000, 40000, 90000], ['Steve Jobs', 30000, 30000]]
+
+
+def donor_names():
+    """ Return a list of donor names """
+    names = []
+    for donor in donation_list:
+        names.append(donor[0])
+    return names
+
+
+def send_thank_you():
+    """
+    Function to get print thank you note based on donor name.
+    :return:
+    """
+    name = input("Enter the full name of the donor:")
+    if name.lower() == 'main menu':
+        return
+    else:
+        if name == 'list':
+            print("Donors are:{}".format(donor_names()))
+            send_thank_you()
+        elif name not in donor_names():
+            # New donor, get his info and print thank you note
+            amount = input("New Donor. Enter the donation amount:")
```

Comment:
cast `input` in float as you take it in so you don't need an extra line to do so